### PR TITLE
String scalar functions: chr

### PR DESCRIPTION
**Pull request by pooja1pathak**
## Summary of the changes / Why this improves CrateDB
Related #9667

## Checklist

 - [x] Added an entry in `CHANGES.txt` for user facing changes
 - [x] Updated documentation & `sql_features` table for user facing changes
 - [x] Touched code is covered by tests
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [x] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

**Comment by mfussenegger** [comment]
ok to test

File: docs/general/builtins/scalar.rst [hunk 1]
```diff
@@ -221,6 +221,26 @@ Returns: ``int``
     +----+--------+
     SELECT 1 row in set (... sec)
 
+.. _scalar_chr:
+
+``chr(int)``
+-----------------
+
+Returns the Character with the given code. For UTF8 the argument is
+treated as a Unicode code point.
```
Comment (mechanomi) [inline docs/general/builtins/scalar.rst:230]:
```suggestion
Returns the Character with the given code. For UTF8 the argument is treated as
a Unicode code point.
```

wrapped to 79 chars

File: docs/general/builtins/scalar.rst [hunk 2]
```diff
@@ -221,6 +221,26 @@ Returns: ``int``
     +----+--------+
     SELECT 1 row in set (... sec)
 
+.. _scalar_chr:
+
+``chr(int)``
+-----------------
```
Comment (mechanomi) [inline docs/general/builtins/scalar.rst:227]:
```suggestion
``chr(int)``
------------
```

fixes underline length

**Comment by mfussenegger** [comment]
@pooja1pathak do you plan to bring this over the finish line? Otherwise we can also pick it up from here and wrap it up.

**Comment by pooja1pathak** [comment]
Yes, I have updated the code as per comment and will update the PR today.

**Comment by pooja1pathak** [comment]
@mfussenegger , I have updated the PR, kindly review and let me know if any further change is required.

**Comment by mfussenegger** [comment]
Thank you again for updating. I made some smaller style changes - I think this looks good now and we can merge it.

**Pull request merged**